database/prefixdb/db_test.go

```diff
@@ -25,6 +26,19 @@ func TestInterface(t *testing.T) {
 	}
 }
 
+func TestSuffix(t *testing.T) {
+	testString := []string{"hello", "world", "a\xff", "\x01\xff\xff\xff\xff"}
+	expected := []string{"hellp", "worle", "b\x00", "\x02\x00\x00\x00\x00"}
+	for i, str := range testString {
+		db := newDB([]byte(str), nil)
+		suf := db.suffix()
+		if !bytes.Equal(*suf, []byte(expected[i])) {
+			t.Fatalf("Expected %s but got %s", expected[i], string(*suf))
+		}
```

Comment:
nit: can we use the `github.com/stretchr/testify/require` library here to align with the other tests?